platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r213/ModelsWithGradleProjectCrossVersionSpec.groovy

```diff
@@ -30,7 +29,6 @@ import org.gradle.tooling.model.gradle.GradleBuild
 import org.gradle.tooling.model.idea.BasicIdeaProject
 import org.gradle.tooling.model.idea.IdeaProject
 
-@ToolingApiVersion('>=3.3')
```

Comment:
❓ doesn't this removal lead to executing e.g. the first 2 tests in this file to be executed with older versions than 3.0?

Reply:
The annotation being removed actually overrode the annotation on the base class. So, in fact, removing it made the test more restrictive as it will run with Tooling API versions 7.0+.